src/Widgets/Presets/DefaultPageNotFoundPreset.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Ceres\Widgets\Presets;
+
+use Ceres\Widgets\Helper\Factories\PresetWidgetFactory;
+use Ceres\Widgets\Helper\PresetHelper;
+use Plenty\Modules\ShopBuilder\Contracts\ContentPreset;
+
+class DefaultPageNotFoundPreset implements ContentPreset
+{
+    public function getWidgets()
+    {
+        /** @var PresetHelper */
+        $preset = pluginApp(PresetHelper::class);
+
+        /** @var PresetWidgetFactory $row */
+        $row = $preset
+            ->createWidget("Ceres::TwoColumnWidget")
+            ->withSetting("customClass", "")
+            ->withSetting("layout", "oneToOne")
+            ->withSetting("layoutTablet", "oneToOne")
+            ->withSetting("layoutMobile", "stackedMobile");
+
+        $row->createChild("first", "Ceres::InlineTextWidget")
+            ->withSetting("text", '<h1>{{ trans("Ceres::Template.pageNotFoundOops") }}</h1><p>{{ trans("Ceres::Template.pageNotFoundPage") }}</p>')
+            ->withSetting("customClass", "")
+            ->withSetting("appearance", "none")
+            ->withSetting("spacing.customPadding", true)
+            ->withSetting("spacing.padding.top.value", 5)
+            ->withSetting("spacing.padding.top.unit", null)
+            ->withSetting("spacing.padding.bottom.value", 0)
+            ->withSetting("spacing.padding.bottom.unit", null)
+            ->withSetting("spacing.padding.left.value", 0)
+            ->withSetting("spacing.padding.left.unit", null)
+            ->withSetting("spacing.padding.right.value", 0)
+            ->withSetting("spacing.padding.right.unit", null);
+
+
+        $row->createChild("first", "Ceres::SeparatorWidget")
+            ->withSetting("customClass", "");
+
+        $row->createChild("first", "Ceres::LinkWidget")
+            ->withSetting("customClass", "")
+            ->withSetting("appearance", "primary")
+            ->withSetting("size", "md")
+            ->withSetting("block", false)
+            ->withSetting("url.type", "external")
+            ->withSetting("url.value", "/");
+
+        $row->createChild("second", "Ceres::InlineTextWidget")
+            ->withSetting("text", '<p class="align-right">404</p>')
```

Comment:
Kann die Klasse nicht auch unter customClass angegeben werden?

Reply:
Theoretisch schon. Dadurch dass sie auf dem p-Tag sitzt, wird sie allerdings vom Inline-Editing berücksichtigt und dort in der Toolbar entsprechend als rechtsbündig angezeigt.